iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/serviceclient/ExportImportTests.java

```diff
@@ -56,7 +55,26 @@
 
     private static RegistryManager registryManager;
 
+    @BeforeClass
     public static void setUp() throws URISyntaxException, InvalidKeyException, StorageException, IOException
+    {
+        if (iotHubConnectionString == null || iotHubConnectionString.isEmpty())
+        {
+            iotHubConnectionString = Tools.retrieveEnvironmentVariableValue(TestConstants.IOT_HUB_CONNECTION_STRING_ENV_VAR_NAME);
+            isBasicTierHub = Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue(TestConstants.IS_BASIC_TIER_HUB_ENV_VAR_NAME));
+            storageAccountConnectionString = Tools.retrieveEnvironmentVariableValue(TestConstants.STORAGE_ACCOUNT_CONNECTION_STRING_ENV_VAR_NAME);
+            isPullRequest = Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue(TestConstants.IS_PULL_REQUEST));
+        }
+
+        // If the environment variable isn't set, then it is likely because it is an android device running these tests
+        // If running these tests on android, a different method will get the environment variables, and then call setUpCommon().
```

Comment:
this shouldn't be required any more, correct?

Reply:
Yep, I'll delete this